Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM registry.svc.ci.openshift.org/openshift/release:golang-1.10 AS builder
-WORKDIR /go/src/github.com/openshift/cluster-ingress-operator
+FROM registry.svc.ci.openshift.org/openshift/release:golang-1.12 AS builder
+WORKDIR /go
 COPY . .
-RUN make build
+RUN make GOPATH="" build
 
 FROM registry.svc.ci.openshift.org/openshift/origin-v4.0:base
```

Comment:
Maybe tangential to this commit, but why are we still use v4.0?

Reply:
v4.2 not existing is the main reason 😆

Reply:
Does v4.1 exist?

Reply:
Nope